debug/list.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
We should add a good example, but that is fine on another PR, @arianaescobar and @makella could help us with that

Reply:
ummmm yeah, if I could get a brief walkthrough of what this is, that would be great :)

Reply:
Ah, sorry, yes. This a new feature to be able to get the list of the features that are on the viewport like: `[{id: 5, name: 'Madrid', pop: 500000}, {id: 6, name: 'Denver', pop:10000000}]`. Of course, the interesting part is to use that to create widgets